includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -2,6 +2,8 @@
 
 class DesignSystemGlobalFooterModel extends WikiaModel {
 	const DEFAULT_LANG = 'en';
+	const WIKI_PRODUCT = 'wikis';
+	const FANDOM_PRODUCT = 'fandoms';
```

Comment:
IMO these constants should follow the THING_TYPE_SUBTYPE format for constants. Although maybe that's not followed at Wikia. So these would be `PRODUCT_WIKI` and `PRODUCT_FANDOM` instead.